libindy/src/services/pool/request_handler.rs

```diff
@@ -285,7 +285,10 @@ impl<T: Networker> RequestSM<T> {
             }
             RequestState::Consensus(mut state) => {
                 match re {
-                    RequestEvent::Reply(_, raw_msg, node_alias, req_id) => {
+                    RequestEvent::Reply(_, raw_msg, node_alias, req_id) |
```

Comment:
In current codebase we have some lack of unit-tests coverage.
@mattraffel please fix it for updated functionality: add something like `request_handler_process_reqnack_event_from_consensus_state_works_for_consensus_reached` but for mixed events. E.g. check what for 4 nodes in pool 1 NACK + 1 REJECT hasn't finished request processing.

Reply:
Probably we should duplicate some test-cases for single and consensus types

Reply:
Added a test case for mixed events.   I can add more if needed.  I do not understand the other request to "duplicate some test-cases".  Not sure what we should be duplicating and changing.  Can you clarify?